<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>productupdater</class>
 <widget class="QDialog" name="productupdater">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame
{
	background-color: rgb(34, 40, 49);
	
}



QDialog
{
	background-color: rgb(238, 238, 238);
}



QLabel
{
	background-color:rgb(238, 238, 238);
	color: rgb(34, 40, 49);
	font: 11pt &quot;DejaVu Sans Mono&quot;;
}

QLineEdit
{

	color:rgb(238, 238, 238);
	background-color: rgb(34, 40, 49);
	border-radius: 9px;
	font: 9pt &quot;DejaVu Sans Mono&quot;;

}


QPushButton
{
	border-radius: 12px;
	background-color: rgb(255, 211, 105);
	color: rgb(34, 40, 49);
	font: 700 11pt &quot;DejaVu Sans Mono&quot;;
	
	
}




</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>460</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>New Stock</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>-60</x>
     <y>0</y>
     <width>700</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 18pt &quot;Ubuntu&quot;;
background-color: rgb(34, 40, 49);
color:rgb(238, 238, 238);
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>18</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Products Update</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>540</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>New Unit Price</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>300</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>New Code</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="errorCurrentCode">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>170</y>
     <width>161</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(238, 238, 238) ;
color: rgb(192, 28, 40);
font: 9pt &quot;DejaVu Sans Mono&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>380</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>New Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="codeIn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>330</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameIn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>410</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="stockIn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>490</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="unitIn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>570</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="updateProduct">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>630</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteProduct">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>220</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearScreen">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>680</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Current Code</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="currentcodeIn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkNewCode">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>330</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkNewName">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>410</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkNewStock">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>490</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkNewUnitPrice">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>570</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line2_2">
   <property name="geometry">
    <rect>
     <x>-60</x>
     <y>270</y>
     <width>1300</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(57, 62, 70);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="errorNewCode">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>330</y>
     <width>151</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(238, 238, 238) ;
color: rgb(192, 28, 40);
font: 9pt &quot;DejaVu Sans Mono&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="updatedSucessfully">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>740</y>
     <width>221</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(238, 238, 238) ;
color: rgb(192, 28, 40);
font: 9pt &quot;DejaVu Sans Mono&quot;;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>clearScreen</sender>
   <signal>clicked()</signal>
   <receiver>unitIn</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>279</x>
     <y>695</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>575</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearScreen</sender>
   <signal>clicked()</signal>
   <receiver>stockIn</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>692</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>492</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearScreen</sender>
   <signal>clicked()</signal>
   <receiver>nameIn</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>699</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>419</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearScreen</sender>
   <signal>clicked()</signal>
   <receiver>codeIn</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>263</x>
     <y>689</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearScreen</sender>
   <signal>clicked()</signal>
   <receiver>errorCurrentCode</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>686</y>
    </hint>
    <hint type="destinationlabel">
     <x>549</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearScreen</sender>
   <signal>clicked()</signal>
   <receiver>currentcodeIn</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>703</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>currentcodeIn</sender>
   <signal>textChanged(QString)</signal>
   <receiver>errorCurrentCode</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>393</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>458</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>codeIn</sender>
   <signal>textChanged(QString)</signal>
   <receiver>errorCurrentCode</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>336</y>
    </hint>
    <hint type="destinationlabel">
     <x>470</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>codeIn</sender>
   <signal>textChanged(QString)</signal>
   <receiver>errorNewCode</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>391</x>
     <y>347</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearScreen</sender>
   <signal>clicked()</signal>
   <receiver>errorNewCode</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>691</y>
    </hint>
    <hint type="destinationlabel">
     <x>563</x>
     <y>345</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearScreen</sender>
   <signal>clicked()</signal>
   <receiver>updatedSucessfully</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>697</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>761</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>currentcodeIn</sender>
   <signal>textChanged(QString)</signal>
   <receiver>updatedSucessfully</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>746</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>codeIn</sender>
   <signal>textChanged(QString)</signal>
   <receiver>updatedSucessfully</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>233</x>
     <y>332</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>744</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
